packages/formvuelate/tests/unit/SchemaRow.spec.js

```diff
@@ -55,4 +55,28 @@ describe('SchemaRow', () => {
 
     expect(wrapper.findAllComponents(SchemaField).length).toBe(2)
   })
+
+  it('doesnt render wrapper elements in slim mode', () => {
+    const wrapper = shallowMount(SchemaRow, {
+      props: {
+        row: [
+          {
+            model: 'FirstName',
+            component: FormText,
+            label: 'First Name',
+            condition: model => false
```

Comment:
Switch this condition to true, or remove it instead
Then lets please switch the test to something clearer like testing that the wrapper contains the two elements intead of a div with two elements inside of it